src/ExpressReceiver.ts

```diff
@@ -82,7 +82,7 @@ export default class ExpressReceiver extends EventEmitter implements Receiver {
     );
     const event: ReceiverEvent = {
       body: req.body as { [key: string]: any },
-      ack: (response: any): void => {
+      ack: async (response: any): Promise<void> => {
```

Comment:
Express's `res.send` is actually synchronous and won't ever error. I believe this still makes sense to force the signature to a promise for other types of receivers.